libraries/AP_HAL_ChibiOS/hwdef/Durandal/hwdef.dat

```diff
@@ -204,6 +204,9 @@ PB0 RSSI_IN ADC1 SCALE(1)
 PC0 VDD_5V_SENS ADC1 SCALE(2)
 PC1 SCALED_V3V3 ADC1 SCALE(2)
 
+# voltage divider 1/(16.9/(33+16.9))
+define HAL_IOMCU_VSERVO_SCALAR 2.952
```

Comment:
Why not actually put the math fully here? If you are worried about < 1mV values then you should just use the higher precision value here inside parens so that the compiler resolves it.

Reply:
Because that would make too much sense when my brain is doing monkey see monkey do matching the other code. Fixed.